cockpit/README.md

```diff
@@ -13,6 +13,18 @@ code is edited.
 
 Development Outside of Vagrant
 ------------------------------
+> Important note for Red Hat developers: When you try to develop cockpit plugin
+outside Red Hat office, then you needn't to do any changes of configuration file.
+In case you are Red Hat developer working in Red Hat office, then please use
+our internal cache of nmpjs repository called Nexus. All you need to do is change
+symbolic link: `rm -f yarn.lock; ln -s yarn.lock.nexus yarn.lock`. There are
+two lock files: `yarn.lock.npmjs` (for community users) and `yarn.lock.nexus` (for
+Red Hat developers). In case you did some big change in `yarn.lock.npmjs` and you
+want to have similar change in `yarn.lock.nexus`, then you can use following
+regul expression:
```

Comment:
Grammar cleanup:
 
An important note for Red Hat developers: If you develop the cockpit
plugin outside of the Red Hat office, no changes to the configuration file are
needed . If you are Red Hat developer working in the Red Hat office, then please use
our internal cache of the nmpjs repository called Nexus. You need to change the
symbolic link: `rm -f yarn.lock; ln -s yarn.lock.nexus yarn.lock`. There are
two lock files: `yarn.lock.npmjs` (for community users) and `yarn.lock.nexus` (for
Red Hat developers). If you make a big change in `yarn.lock.npmjs` and you
want to have similar change in `yarn.lock.nexus`, then you should use following
regular expression:

Reply:
Thanks!

Reply:
Thanks